src/node.cc

```diff
@@ -1135,8 +1143,8 @@ InitializeOncePerProcessInternal(const std::vector<std::string>& args,
 
     {
       std::string extra_ca_certs;
-      if (credentials::SafeGetenv("NODE_EXTRA_CA_CERTS", &extra_ca_certs))
-        crypto::UseExtraCaCerts(extra_ca_certs);
+      GetFromDotenvFileOrSystem("NODE_EXTRA_CA_CERTS", &extra_ca_certs);
```

Comment:
This changes the behavior. Previously, we were calling `UseExtraCaCerts` conditionally. Now, we don't. Can you make sure we don't change the behavior?